packages/form-field/src/FormField/FormField.tsx

```diff
@@ -56,16 +49,20 @@ export const FormField = forwardRef<HTMLDivElement, FormFieldProps>(
     }: FormFieldProps,
     fwdRef,
   ) => {
-    const { theme } = useDarkMode();
     const baseFontSize = useUpdatedBaseFontSize(baseFontSizeProp);
     const fontStyles = getFontSize({ baseFontSize, size });
 
-    const { labelId, descriptionId, errorId, inputId, inputProps } =
+    const { labelId, descriptionId, feedbackId, inputId, inputProps } =
       useFormFieldProps({ label, description, state, id, disabled, ...rest });
 
-    const isErrorState = state === FormFieldState.Error;
-    const isSuccessState = state === FormFieldState.Valid;
-    const showStateFeedback = (isErrorState || isSuccessState) && !disabled;
+    const formFieldFeedbackProps = {
+      disabled,
+      errorMessage,
+      id: feedbackId,
+      size,
+      state,
+      successMessage,
+    } as const;
```

Comment:
Maybe a topic for later, but it may be worth considering adding most of these props to a FormFieldContext?

Reply:
Actually, most of these are already on the provider. Could we pull these from the provider in `FormFieldFeedback` instead of passing them as props?

Reply:
(That does make it harder to test though, to take it or leave it)

Reply:
I think I'll leave it as-is for now since `FormFieldFeedback` will at times be used outside of a `FormFieldContext`